docker/dependencies.R

```diff
@@ -9,6 +9,7 @@ library(eds)
 library(eisaR)
 library(ensembldb)
 library(fishpond)
+library(ggforce)
 library(ggplot2)
```

Comment:
I'm not sure how `ggplot2` got into this file, but we should not need it as `ggplot2` is already a dependency of required packages. We also made `fishpond` a required dependency at some point, so it should be installed as well without the need to be in this file).
```suggestion
library(ggforce)
```